src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2107,6 +2107,26 @@
           "type": "string"
         },
 
+        "organization": {
+          "description": "The organization or company that produced the tool.",
+          "type": "string"
+        },
+
+        "product": {
+          "description": "A product suite to which the tool belongs.",
+          "type": "string"
+        },
+
+        "shortDescription": {
+          "description": "A brief, multiformat description of the tool.",
```

Comment:
>multiformat [](start = 35, length = 11)

I don't think the word "multiformat" is needed in the description. Just "A brief description of the tool".